### ISSUE TITLE
[Feature request] Display awaiting BTC TX

### ISSUE THREAD

**Issue by CmdCrypto** [breez/breezmobile#752]
Would be a **huge** UI Improvement to display awaiting incoming BTC TX although not yet fully confirmed (Example: Like Samurai Wallet).

Example: Incoming BTC Transaction about XX.XXX BTC (X/3 confirmed)

**Comment by kingonly** [comment]
You actually have the information in the Receive via BTC Address screen, but it makes sense to add it to the payments list as well. We'll look into it.

**Comment by CmdCrypto** [comment]
Well, not exactly - on the mains screen I get "Breez is waiting for Bitcoin transfer to be confirmed...." - no info about how many, when, how many confs are missing, tx id. You are thinking like a developer ;-) - I am like a standard user :-p

**Comment by CmdCrypto** [comment]
Saw your comment on twitter https://twitter.com/Breez_Tech/status/1488838309614919682 - totally unacceptable.

**Comment by tlindi** [comment]
>You actually have the information in the Receive via BTC Address screen, but it makes sense to add it to the payments list as well. We'll look into it.

I think the exclamation mark top right would do also work for me. Details by clicking.

**Comment by kingonly** [comment]
What this has to do with this issue? And with all due respect to your critique, you are clearly unaware of the work that is being done on enterprise reliability, so for me saying stuff like "lightning was introduced in 2015" or "unacceptable" doesn't provide value. If you're really interested in the success of Lightning, this isn't the way.

**Comment by CmdCrypto** [comment]
It provides more value than your comments. Good luck with your app but if you take critics this way you will never ever succeed - I am the owner of 2 big LN Nodes for several years now, my nodes did not go a single sec offline bc of a simple disk failure. And if they would do and my users get upset I would do everything I could to assure that this will not happen again instead of pissing those users off. As I said, good luck, you will need it.

**Comment by kingonly** [comment]
I don't think you know me or Breez very well. You already said I'm thinking like a developer and don't know how to handle critique. Where in fact, I'm not a developer and I accepted your feedback on the btc tx.

Re enterprise reliability: if you run lnd, you should be aware of the missing features of fail overs and reliability. We do our best with the existing constraints, but some of the stuff are missing (load balancing of same node for example). We're constantly trying to improve and we've gone a long way with the help of our users. I'm sorry if I offended you, that wasn't the intention. The feedback you provided on Twitter (not here) wasn't very productive. I delete the tweet. I encourage you to take a look at Breez and the work we've done over the years to advance Lightning. Now take a look how your tweets read from our standpoint.
